FoodOrderWebApi/Model/Product.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace FoodOrderWebApi.Model
+{
+    public class Product
```

Comment:
Should restaurants be able to temporarily pull a product from the menu (make it disabled)?